pages/pvp.md

```diff
@@ -14,23 +14,33 @@
 
 ## Welcome to SC2Aiur's general guide to Protoss vs Protoss
 
-*General description of PvP*
+PvP is the only matchup where you will delay your expansion in favor of having more units early. The general game plan starts with two gateways, from which you build either two adepts, two stalkers, or one stalker and a sentry. From there, you can be aggressive or defensive with your starting units, and decide whether to commit to aggression, expand go any of the three tech routes, as robo, twilight into blink or stargate are all viable options. The main mid game compositions to build towards are generally either Chargelot-Immortal-Archon based (CIA), or heavy on the blink stalkers, with disruptors being excellent in either composition.
```

Comment:
Here's my first actual content concern:

```suggestion
PvP is the only matchup where you will **delay your expansion in favor of having more units early**. The general game plan starts with **two gateways**, from which you build either two adepts, two stalkers, or one stalker and a sentry. From there, you can be aggressive or defensive with your starting units, and decide whether to commit to aggression, expand, or go any of the three tech routes, as robo, twilight into blink or stargate are all viable options. The main mid game compositions to build towards are generally either Chargelot-Immortal-Archon based (CIA), or one based on [blink stalkers for mobility with disruptors for bursty firepower](posts/blinkdisruptor-era-pvp/).
```

I haven't seen many games with CIA having disruptors mixed in. I'd say it's counterproductive, because your zealots die to your own novas a lot of the time. I'd agree that after an initial period of CIA aggression, a transition to stalker/disruptor is often started. Whatcha think?